### PR TITLE
[#10559] Pages with rich text editors cannot be displayed

### LINKED ISSUE #10559: Pages with rich text editors cannot be displayed
Chrome, Win10

Steps:
1. Open a fresh browser
1. Login as an instructor
1. Click on the `Sessions` link in the top navigation bar

Problem: nothing happens. Same for the `Search` link. But you can go to the page by editing the browser URL. Furthermore, the problem goes away after visiting that page once.

Let me know if cannot reproduce.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #10559 

Cause: one of the libraries recently added, `ng-in-viewport`, has `new Function` as part of its code, which is blocked under `unsafe-eval` CSP policy.

Solution:
- There is a very recent version that did not use any `unsafe-eval`-breaking code; use that version
- Change the local HTML file to also exclude `unsafe-eval` so that this kind of problem can be detected earlier

I have raised the issue to the library author: https://github.com/k3nsei/ng-in-viewport/issues/333

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "intersection-observer": "^0.11.0",
     "moment-timezone": "^0.5.31",
     "ng-dynamic-component": "^6.1.0",
-    "ng-in-viewport": "^6.1.1",
+    "ng-in-viewport": "6.1.0",
```
Comment (xpdavid) [inline package.json:42]: Downgrading?

**Comment by damithc** [comment]
Which kind of test should have caught this regression? Was it left out by mistake or that test is yet to be added?

File: package.json (hunk 1 above)
Comment (wkurniawan07) [reply package.json]: Was. The problem has been fixed.

**Comment by wkurniawan07** [comment]
@damithc if the CSP policy between both `npm run start` and `npm run build` is the same (which has been fixed in this very PR), this problem would have been caught earlier.

**Comment by damithc** [comment]
> @damithc if the CSP policy between both `npm run start` and `npm run build` is the same (which has been fixed in this very PR), this problem would have been caught earlier.

Got it. 👍

**Pull request merged**